02-csv-vector-raster-plotting.Rmd

```diff
@@ -91,20 +94,20 @@ library(raster)
 
 ```
 
-##Reading in our CSV
+##Reading in a CSV
 
 The `HARV_PlotLocations.csv` file contains x,y locations for some plots that 
-(NEON)[http://www.neoninc.org] field crew are sampling vegetation and other 
+(NEON)[http://www.neoninc.org] field crews are sampling vegetation and other 
 ecological metrics. We would like to create a map that includes the locations of 
 these plot locations. We also would like to export the data in a `shapefile` 
 format to share with our colleagues. This shapefile can also be imported into any
-`GIS` software.
+GIS software.
 
 To begin, let's read in the `csv` file that contains the x,y coordinate locations.
 
 ```{r read-csv }
 
-#Leah's notes
+#set working directory to where you have stored the data
```

Comment:
ok - referring to issue https://github.com/data-lessons/NEON-R-Spatial-Vector/issues/10 -- let's address the working directory issue via that issue and make a call. i'm inclined to remove the setwd code throughout but maybe have something commented that says - be sure to set your working directory. leave as is and address via the issue. :)